pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -115,10 +115,14 @@ def my_func(self):
             """This is a docstring.
 
             :raises RuntimeError: Always
-            :raises NameError: Never
+            :except NameError: Never
```

Comment:
Oh, but this is not a bug. In your example, extract_node is extracting the function node, not a raise node, so when calling visit_raise, you will have problems as this one. You probably should call visit_functiondef instead.